splunk/splunk-config.yml

```diff
@@ -0,0 +1,77 @@
+splunk:
+  conf:
+    indexes:
+      directory: /opt/splunk/etc/apps/search/local
+      content:
+        metrics:
+          coldPath: $SPLUNK_DB/metrics/colddb
+          datatype: metric
+          homePath: $SPLUNK_DB/metrics/db
+          maxTotalDataSizeMB: 512000
+          thawedPath: $SPLUNK_DB/metrics/thaweddb
+        ethereum:
+          coldPath: $SPLUNK_DB/ethereum/colddb
+          datatype: event
+          homePath: $SPLUNK_DB/ethereum/db
+          maxTotalDataSizeMB: 512000
+          thawedPath: $SPLUNK_DB/ethereum/thaweddb
+        logs:
+          coldPath: $SPLUNK_DB/logs/colddb
+          datatype: event
+          homePath: $SPLUNK_DB/logs/db
+          maxTotalDadtaSizeMB: 512000
+          thawedPath: $SPLUNK_DB/logs/thaweddb
+    inputs:
+      directory: /opt/splunk/etc/apps/search/local
+      content:
+        "udp://8125":
+          connection_host: ip
+          index: metrics
+          sourcetype: statsd
+        "logs://cadivsor":
+          index: logs
+          sourcetype: cadvisor
+          disabled: 0
+        "logs://node1":
+          index: logs
+          sourcetype: node1
+          disabled: 0
+        "logs://txmanager1":
+          index: logs
+          sourcetype: txmanager1
+          disabled: 0
+        "logs://node2":
+          index: logs
+          sourcetype: node2
+          disabled: 0
+        "logs://txmanager2":
+          index: logs
+          sourcetype: txmanager2
+          disabled: 0
+        "logs://node3":
+          index: logs
+          sourcetype: node3
+          disabled: 0
+        "logs://txmanager3":
+          index: logs
+          sourcetype: txmanager3
+          disabled: 0
+        "logs://node4":
+          index: logs
+          sourcetype: node4
+          disabled: 0
+        "logs://txmanager4":
+          index: logs
+          sourcetype: txmanager4
+          disabled: 0
+        "logs://cakeshop":
+          index: logs
+          sourcetype: cakeshop
+          disabled: 0
```

Comment:
What happens if there are more/less than 4 nodes?

Reply:
ah, i believe this was left over from a previous iteration. i removed, but will confirm.

Reply:
confirmed - not needed.